test/test-project/src/test/editor/diffEditor-test.ts

```diff
@@ -0,0 +1,38 @@
+import { expect } from 'chai';
+import { TextEditor, EditorView, Workbench, DiffEditor, QuickOpenBox } from "vscode-extension-tester";
+
+describe('DiffEditor', async () => {
+    let editor: DiffEditor;
+
+    before(async function() {
+        this.timeout(15000);
+        await new Workbench().executeCommand('File: New Untitled File');
```

Comment:
I have to call this instead of "File: New File", otherwise the untitled file is not created (all tests that use "File: New File" failed on my machines because of this)